### ISSUE TITLE
[Deque Analysis] Browse Community/Collection "serious" issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1177]
> Serious: This issue results in serious barriers for people with disabilities, and will partially prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration as a result. Issues falling under this category are major problems, and remediation should be a priority. 

## Deque Analysis Summary

Browse by Community/Collection page has a total of 2 serious issues (not including color contrast which are handled in other tickets):

1. (ID 469376) ~~"The element appears and functions like a treeview but is missing required ARIA role(s) and/or attribute(s)" (on the entire page structure).~~
    * ~~Fix this issue by doing ONE of the following: (1) Follow the ARIA accordion/toggle design pattern including expected keyboard interactions: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/ OR, (2) Follow the full ARIA treeview design pattern including expected keyboard interactions: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/~~
    * ~~Also see https://dequeuniversity.com/class/custom-widgets2/examples/expand-collapse and/or https://dequeuniversity.com/class/custom-widgets2/examples/tree~~
    * NOT REPRODUCIBLE IN 7.6.1.  As of that release, this appears to be a [TreeView](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/) with valid "role" and ARIA attributes.  Unsure when this was fixed.
2. ~~(ID 469379) "The element functions as if it has two roles." (this occurs on each of the expandable Community links).~~ FIXED by #2251 and #2447
     * Additional comment: "The element functions in two ways, on activating with mouse the element gets expanded and on activating the link with keyboard it is redirected to its respective page. This is confusing."
     * Only fix is to redesign the user interface so that each element has only *one role or function* (and must behave the same way when selected via mouse or keyboard)

Full list of issues is viewable at (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=50&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=3&filter%5Btype%5D=issue&filter%5Bpage_number%5D=3

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by tdonohue** [comment]
Closing, as all issues appear to be fixed as of 7.6.1.  See notes in description above.
